programs/protocol/src/data/name.rs

```diff
@@ -0,0 +1,10 @@
+use anchor_lang::prelude::{
+    borsh::{BorshDeserialize, BorshSerialize},
+    *,
+};
+
+#[zero_copy]
+#[derive(Debug, Default, PartialEq, BorshSerialize, BorshDeserialize, Eq)]
+pub struct Name {
+    pub value: [u32; 16], // 64 characters
```

Comment:
isnt better to store u16 like utf8 in rust?

Reply:
not sure why but it would not compile